### DISCUSSION TITLE
One Jaeger URL to show QA and Prod data in the same page

### DISCUSSION THREAD

**Discussion by winshenting** [jaegertracing/jaeger#3131] [General]
Dear all,
I want Jaeger Portal shows QA and Prod Data in the same page. 
I config qa index-prefix: micro-qa; cm index-prefix: micro-cm
I config query index-prefix: micro
But it does not work. it does not show both qa and cm in the same portal page.
What shall I do? 
`  collector:
    maxReplicas: 10
    autoscale: true
    num-workers: 50
    queue-size: 2000
    config: {}
    options:
      log-level: debug
    resources:
      requests:
        cpu: '2'
        memory: 2Gi
      limits:
        cpu: '4'
        memory: 4Gi
  ingester:
    config: {}
    options: {}
    resources: {}
  ingress:
    annotations:
      kubernetes.io/ingress.class: internal-nginx
      nginx.ingress.kubernetes.io/backend-protocol: HTTP
      nginx.ingress.kubernetes.io/proxy-body-size: 10m
    labels: {}
    hosts:
      - jaeger.qa.com
    path:
      /
    openshift: {}
    options: {}
    resources: {}
    security: none
  query:
    options:
      log-level: debug
      es:
        index-prefix: micro
    resources:
      requests:
        cpu: '2'
        memory: 2Gi
      limits:
        cpu: '4'
        memory: 4Gi
    options:
      es:
        create-index-templates: true
        max-num-spans: 10000
        max-span-age: 72h0m0s
        timeout: 120s
        index-prefix: micro-qa
 `

**Comment by jpkrohling** [comment]
I think this has now been answered three times :-) In short: the Jaeger Query cannot at this point get data from multiple backing storages.
